cmake/kokkos_compiler_id.cmake

```diff
@@ -137,6 +137,17 @@ IF(KOKKOS_CXX_COMPILER_ID STREQUAL Cray OR KOKKOS_CLANG_IS_CRAY)
   ENDIF()
 ENDIF()
 
+IF(KOKKOS_CXX_COMPILER_ID STREQUAL Fujitsu)
+  # SET Cray's compiler version.
```

Comment:
I assume you copied from above and forgot to update or remove the comment

Reply:
yeah ...

Reply:
fixed